chromium_src/components/history/core/browser/history_backend.cc

```diff
@@ -0,0 +1,82 @@
+/* Copyright (c) 2022 The Brave Authors. All rights reserved.
```

Comment:
It's now 2023 :champagne:

Reply:
Indeed :) . 
That is hard when PR is created during the year change :) .

Previous years I didn't faced this kind of situation.
Right now the year corresponds the date of the commit:
![image](https://user-images.githubusercontent.com/24739341/210596893-49cdae51-98f0-4ba6-a8b0-1b92952a1a35.png)

But the actual year when it will reach the master will be 2023. 

So which year should be chosen: from the date of commit or from the date of actual merge?

Reply:
After rebase the commit date was overwritten in GitHub but `git log` still displays 2022 for the commit. In anyway I had updated the year, thanks.

Reply:
> So which year should be chosen: from the date of commit or from the date of actual merge?

Good question. I'd probably go with the year of the merge since that's closer to when the code is "published". But it would probably be fine either way.

It's really just an opportunity for me to say "happy new year" ;)